tests/lib/ocpp/v201/test_device_model.cpp

```diff
@@ -11,29 +14,52 @@ namespace v201 {
 
 class DeviceModelTest : public ::testing::Test {
 protected:
-    const std::string DEVICE_MODEL_DATABASE = "./resources/unittest_device_model.db";
-    std::unique_ptr<DeviceModel> dm;
+    DeviceModelTestHelper device_model_test_helper;
+    DeviceModel* dm;
     const RequiredComponentVariable cv = ControllerComponentVariables::AlignedDataInterval;
+    const std::map<int32_t, int32_t> evse_connector_structure{{1, 1}, {2, 1}};
 
-    void SetUp() override {
-        dm =
-            std::make_unique<DeviceModel>(std::move(std::make_unique<DeviceModelStorageSqlite>(DEVICE_MODEL_DATABASE)));
-    }
-
-    void TearDown() override {
-        // reset the value to default
-        dm->set_value(cv.component, cv.variable.value(), ocpp::v201::AttributeEnum::Actual, "10", "test");
+    DeviceModelTest() : device_model_test_helper(), dm(device_model_test_helper.get_device_model()) {
     }
 };
 
+TEST_F(DeviceModelTest, test_allow_zero2) {
```

Comment:
nit: `test_allow_zero2` defined before `test_allow_zero` looks a little bit odd. A name like `test_min_limit_and_allow_zero` is probably better